src/App.js

```diff
@@ -1,23 +1,86 @@
-import logo from './logo.svg';
 import './App.css';
+import BoxColour from './components/BoxColour/BoxColour';
+import CreditCard from './components/CreditCard/CreditCard';
+import DriverCard from './components/DriverCard/DriverCard';
+import Greetings from './components/Greetings/Greetings';
+import IdCard from './components/IdCard/IdCard';
+import LikeButton from './components/LikeButton/LikeButton';
+import Random from './components/Random/Random';
+import Rating from './components/Rating/Rating';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div
+      className="App"
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        gap: '20px',
```

Comment:
It is better not to use inline styles, let's avoid it as much as possible. In a large project it is not maintainable to write the styles like this.